lab-carolina/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "lab-01-modular-patterns",
+  "version": "1.0.0",
+  "description": "![CF](assets/shield-32x32.png) Lab 01: Modular Patterns and Testing ===",
+  "main": "index.js",
```

Comment:
You have the 'entry point' defined here as `index.js` but there's no corresponding file. You should create an `index.js` in the same directory (root level) as the `package.json`, which should require in your module(s). You can then run `node index.js` to execute the functionality.